crates/wal-protocol/Cargo.toml

```diff
@@ -10,7 +10,7 @@ publish = false
 [features]
 default = ["serde"]
 serde = ["dep:serde", "enum-map/serde", "bytestring/serde", "restate-invoker-api/serde"]
-options_schema = ["dep:schemars"]
```

Comment:
I think that we might actually be able to remove this feature.

Reply:
Additionally, we should change https://github.com/restatedev/restate/blob/41e606e6c2c6535134f0e7ee5d6bd70ce12b924a/crates/wal-protocol/src/lib.rs#L88 to `#[cfg_attr(feature = "serde", serde(default))]`.

Reply:
fixed, PTAL